### DISCUSSION TITLE
Will CoralTPU work with -tensorrt version

### DISCUSSION THREAD

**Discussion by The-NinthWave** [blakeblackshear/frigate#9467] [Ask A Question] [answered]
I was running Frigate with TensorRT as a detector.
I just installed the CoralTPU
Given that I have modified the container and the config.yaml, is Frigate +Coral supposed to work with the same image tagged -tensorrt ?

**Comment by NickM-27** [comment, marked as answer]
There's no specific reason why it wouldn't but there's also no reason to run -tensorrt if you're using a coral

**Comment by The-NinthWave** [comment]
With my Nvidia P600, I had an inference of 10.71 ms
```
#model:
#  path: /trt-models/yolov4-tiny-416.trt
#  input_tensor: nchw
#  input_pixel_format: rgb
#  width: 416
#  height: 416

```

With the Coral, I am down at 6,75 ms
